upstairs/src/live_repair.rs

```diff
@@ -3759,13 +3698,13 @@ pub mod repair_test {
 
         let eid = 1;
         ds.extent_limit[1] = Some(eid.try_into().unwrap());
-        let ddef = up.ddef.lock().await.get_def().unwrap();
-        let impacted_blocks = extent_to_impacted_blocks(&ddef, eid);
 
         // Upstairs "guest" work IDs.
         let gw_r_id: u64 = 1;
-        let r_id = 1002;
-        let deps = vec![1000, 1001];
+        let (repair_ids, mut deps) = ds.get_repair_ids(eid);
+        assert!(deps.is_empty());
+        deps.push(1000); // TODO(matt) this isn't testing useful stuff?
```

Comment:
Should there be a follow up issue, or rewrite this test?

Reply:
Upon further thought, it does have some use: it's checking that `create_and_enqueue_reopen_io` correctly uses the dependencies that you pass in.  I'll remove the comments here (and below).